mailscanner/functions.php

```diff
@@ -72,7 +72,12 @@
 } else {
     $lang = require __DIR__ . '/languages/' . LANG . '.php';
 }
-
+if (!defined('USE_ANTIALIASING') || !USE_ANTIALIASING) {
```

Comment:
just change this line with the following and the new config entry is not needed:
```php
if (!function_exists('imageantialias')) {
```